packages/taco/examples/ecdsa-condition.ts

```diff
@@ -0,0 +1,116 @@
+import { ECDSACondition } from '../src/conditions/base/ecdsa';
+
+// Example of creating an ECDSA condition
+console.log('=== ECDSA Condition Example ===');
+
+// Example 1: Basic ECDSA condition with static values
+const basicECDSACondition = new ECDSACondition({
+  message: 'Hello, world!',
+  signature: '304402206e41b6b588af8a85c9a9b2e8b4e2b6e1b3c4d5e6f7a8b9c0d1e2f3a4b5c6d7e8f9a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5e6f7a8b9c0d1e2f3a4b5c6d7e8f9',
+  verifyingKey: '04a34b99f22c790c4e36b2b3c2c35a36db06226e41c692fc82b8b56ac1c540c5bd5b8dec5235a0fa8722476c7709c02559e3aa73aa03918ba2d492eea75abea235',
+  curve: 'SECP256k1',
+});
+
+console.log('Basic ECDSA Condition:');
+console.log(JSON.stringify(basicECDSACondition.toObj(), null, 2));
+
+// Example 2: ECDSA condition with context variables
+const dynamicECDSACondition = new ECDSACondition({
+  message: ':userMessage',
+  signature: ':userSignature',
+  verifyingKey: '04a34b99f22c790c4e36b2b3c2c35a36db06226e41c692fc82b8b56ac1c540c5bd5b8dec5235a0fa8722476c7709c02559e3aa73aa03918ba2d492eea75abea235',
+  curve: 'SECP256k1',
+});
+
+console.log('\nDynamic ECDSA Condition (with context variables):');
+console.log(JSON.stringify(dynamicECDSACondition.toObj(), null, 2));
+
+// Example 3: ECDSA condition with different curve
+const p256ECDSACondition = new ECDSACondition({
+  message: 'Authenticated message',
+  signature: ':ecdsaSignature',
+  verifyingKey: '04a34b99f22c790c4e36b2b3c2c35a36db06226e41c692fc82b8b56ac1c540c5bd5b8dec5235a0fa8722476c7709c02559e3aa73aa03918ba2d492eea75abea235',
+  curve: 'NIST256p',
+});
+
+console.log('\nECDSA Condition with P-256 curve:');
+console.log(JSON.stringify(p256ECDSACondition.toObj(), null, 2));
+
+// Example 4: Using defaults for message and signature
+const defaultECDSACondition = new ECDSACondition({
+  message: ':ecdsaMessage', // Default context variable
+  signature: ':ecdsaSignature', // Default context variable
+  verifyingKey: '04a34b99f22c790c4e36b2b3c2c35a36db06226e41c692fc82b8b56ac1c540c5bd5b8dec5235a0fa8722476c7709c02559e3aa73aa03918ba2d492eea75abea235',
```

Comment:
If using the defaults, you should be able to do the following?

```suggestion
const defaultECDSACondition = new ECDSACondition({
  verifyingKey: '04a34b99f22c790c4e36b2b3c2c35a36db06226e41c692fc82b8b56ac1c540c5bd5b8dec5235a0fa8722476c7709c02559e3aa73aa03918ba2d492eea75abea235',
```

or were you trying to be explicit about the defaults?